skipchain/struct.go

```diff
@@ -1120,15 +1122,31 @@ func (db *SkipBlockDB) GetProof(sid SkipBlockID) (sbs []*SkipBlock, err error) {
 		sbs = append(sbs, sb)
 
 		for sb != nil && len(sb.ForwardLink) > 0 {
-			id := sb.ForwardLink[len(sb.ForwardLink)-1].To
-			sb, err = db.getFromTx(tx, id)
+			// In some cases, a forward-link could be stored before the
```

Comment:
Is this an unrelated change? (no big deal, just trying to understand the motivation here)

Reply:
Related to Jenkins stability and it also makes sense. Returning an error if the very latest block is not yet fully stored is wrong because we can send the previous state as the next one is not yet ready.